packages/astro/src/core/config/schema.ts

```diff
@@ -284,6 +285,10 @@ export const AstroConfigSchema = z.object({
 					}),
 				)
 				.default([]),
+			experimentalLayout: z.enum(['responsive', 'fixed', 'full-width', 'none']).optional(),
```

Comment:
A single source of truth for this would be great, you can add it in const, derive the type from it and also use it here

Reply:
*cries in single source of truth for the config types* ahem it would, yes